Doppler.AccountPlans.Test/IntegrationTest1.cs

```diff
@@ -18,10 +18,9 @@ public IntegrationTest1(WebApplicationFactory<Startup> factory)
         }
 
         [Theory]
+        [InlineData("/WeatherForecast", HttpStatusCode.OK, "application/json; charset=utf-8")]
         [InlineData("/swagger", HttpStatusCode.Moved, null)]
         [InlineData("/swagger/index.html", HttpStatusCode.OK, "text/html; charset=utf-8")]
-        [InlineData("/robots.txt", HttpStatusCode.OK, "text/plain")]
-        [InlineData("/favicon.ico", HttpStatusCode.OK, "image/x-icon")]
```

Comment:
`favicon.ico` and `robots.txt` should work, why did you remove them?

Reply:
ok, I was using doppler-users-api as an example to follow and they weren't there, that's why I removed them

Reply:
Oh, @javierburger please restore them in your API.

Reply:
ok